fedimint-rocksdb/src/lib.rs

```diff
@@ -51,6 +51,27 @@ impl From<RocksDb> for rocksdb::OptimisticTransactionDB {
     }
 }
 
+// When finding by prefix iterating in Reverse order, we need to start from
+// "prefix+1" instead of "prefix", using lexicographic ordering. See the tests
+// below
+fn next_prefix(prefix: &[u8]) -> Result<Vec<u8>> {
```

Comment:
This is kind of a `increment_bytes` thing, no? Not even prefix specific.

Reply:
Not sure, what `increment_bytes`?

Reply:
`fn increment_bytes(prefix: &[u8])`? Or did we misunderstand each other? :D

Reply:
Oh, I think it's weird and specific enough to be called `next_prefix` instead of something more generic.